### PR TITLE
Valley Hale Stuff

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Adds an Elyran third party ship, changed the sector availabilities for a lot of away sites and ships. Should ideally be merged before we go to the Valley Hale, as it won't have any content if we don't. Also, at the request of Caelphon, the Elyran ship should be blacklisted from spawning until an event happening in the near future.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

File: maps/away/generic/civilian_station.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "civilian station"
 	description = "A modestly-sized independently-owned civilian space station. Many of these exist all throughout inhabited space, offering a place to rest, food to eat, shopping, and refueling - part mall, part motel. This one appears to have been active in the region long before Biesel took control, and an information lookup indicates that it is operated by a small company that is Solarian in origin. This one's transponder says it's open for business!"
```
Comment (listerla) [inline maps/away/generic/civilian_station.dm:3]: If available in New Ankara, probably shouldn't remain fluffed as Solarian in origin.

File: maps/away/ships/elyra.dm [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+/obj/effect/shuttle_landmark/elyran_strike_craft/transit
+	name = "In transit"
+	landmark_tag = "nav_transit_elyran_strike_craft"
+	base_turf = /turf/space/transit/south
+
+//shuttle stuff
+/obj/effect/overmap/visitable/ship/landable/elyran_shuttle
+	name = "Elyran Naval Shuttle"
+	desc = "Easily mistaken for a Wisp-class (Which it is almost identical to), this shuttle is in fact an Elyran Dromedary-class, which is an unlicensed copy of the Hephaestus Wisp-class, and one of the designs slated for retirement in the ongoing military modernization program. Hephaestus Industries has attempted to sue the Elyran government many times for the navy’s unlicensed production of its intellectual property, and Elyran courts have, unsurprisingly, always ruled in favor of the government - much to the frustration of the megacorporation, which has no other method of recourse. An inefficient design of ultra-light shuttle. Its only redeeming features are the extreme cheapness of the design and the ease of finding replacement parts. This one’s transponder identifies it as an Elyran Naval Infantry recovery shuttle."
```
Comment (listerla) [inline maps/away/ships/elyra.dm:68]: 'Which' shouldn't be capitalized in the parentheses; not a complete sentence, just a fragment.

Comment (TheDanseMacabre) [reply maps/away/ships/elyra.dm:68]: Addressed

File: maps/away/generic/civilian_station.dm (hunk 1 above)
Comment (TheDanseMacabre) [reply maps/away/generic/civilian_station.dm:3]: Addressed

**Review by Alberyk — APPROVED** [review]

File: maps/away/generic/bigderelict.dm [hunk 3]
```diff
@@ -2,7 +2,7 @@
 	name = "large derelict"
 	description = "A very large derelict station. According to the starmap, it shouldn't exist."
 	suffix = "generic/bigderelict.dmm"
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE)
+	sectors = list(SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ)
```
Comment (Alberyk) [inline maps/away/generic/bigderelict.dm:5]: No reason to remove tau ceti and romanovich from the list.

Comment (TheDanseMacabre) [reply maps/away/generic/bigderelict.dm:5]: Think this was an accident on my part

**Review by NonQueueingMatt — APPROVED** [review]

File: maps/away/generic/civilian_station.dm [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
 	suffix = "generic/civilian_station.dmm"
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
+	sectors = list(SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
```
Comment (Alberyk) [inline maps/away/generic/civilian_station.dm:5]:
```suggestion
	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
```

Comment (TheDanseMacabre) [reply maps/away/generic/civilian_station.dm:5]: The civilian station is Solarian in origin and not owned by the SCC, so a different version would be needed for Tau Ceti and the Romanovich. I will potentially make one should we ever end up going back there.

**Pull request merged**